tests/queries/0_stateless/03363_hive_style_partition.sql

```diff
@@ -2,13 +2,13 @@
 
 SET allow_suspicious_low_cardinality_types=1;
 
-DROP TABLE IF EXISTS t_03363_parquet, t_03363_csv, s3_table_half_schema_with_format;
```

Comment:
The diff here includes:

1. new tests
2. renaming table names & s3 locations. The reason for that is that `hive` strategy uses the table root as a prefix for reading, making the reading path something like `table_root**.parquet`. This caused conflicts for `t_03363` and `t_03363_something_else`.